.github/workflows/release-checksums.yml

```
@@ -85,7 +93,7 @@
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          name: ${{ env.RELEASE_NAME }}
+          name: ${{ steps.release-info.outputs.release-name }}
```

actionlint (1.7.4)

property "release-info" is not defined in object type {}

(expression)